apps/wallet/src/ui/app/redux/slices/sui-objects/Coin.ts

```diff
@@ -23,6 +19,7 @@ export const DEFAULT_GAS_BUDGET_FOR_SPLIT = 1000;
 export const DEFAULT_GAS_BUDGET_FOR_MERGE = 500;
 export const DEFAULT_GAS_BUDGET_FOR_TRANSFER = 100;
 export const DEFAULT_GAS_BUDGET_FOR_TRANSFER_SUI = 100;
+export const DEFAULT_GAS_BUDGET_FOR_PAY = 100;
```

Comment:
per our discussion on the other slack thread, shall we change it to a higher value?
after flat Move VM change, some other budgets will be 10k btw https://github.com/MystenLabs/sui/pull/4607/files#diff-10ade975eef79240b8f0304f961422f530f97c4b289645f0d005f7d8be5080e8